### PR TITLE
Update blame configuration.

### PR DESCRIPTION
**Pull request by nohwnd**
Add example of hang dump and crash dump blame configuration.

### CODE REVIEW

**Comment by jborsecnik** [comment]
@Mikejo5000, 

Can you review the proposed changes? 

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/test/configure-unit-tests-by-using-a-dot-runsettings-file.md [hunk 1]
```diff
@@ -214,9 +214,39 @@ To customize any other type of diagnostic data adapters, use a [test settings fi
...
 
-```xml
-<DataCollector friendlyName="blame" enabled="True">
-</DataCollector>
+You can run blame in 3 different modes: 
+- Enabling just the sequence file, but not collecting dumps.
+- Enabling crash dump, to create a dump when testhost crashes.
+- Enabling hang dump, to create a dump when test does not finish before given timeout.
+
+The xml configuration should be placed directly into `<RunSettings>` node:
```
Comment (Mikejo5000) [inline docs/test/configure-unit-tests-by-using-a-dot-runsettings-file.md:222]:
```suggestion
The XML configuration should be placed directly into `<RunSettings>` node:
```

**Comment by Mikejo5000** [comment]
#sign-off Thanks, @nohwnd !

File: docs/test/configure-unit-tests-by-using-a-dot-runsettings-file.md [hunk 2]
```diff
@@ -214,9 +214,39 @@ To customize any other type of diagnostic data adapters, use a [test settings fi
 
 This option can help you isolate a problematic test that causes a test host crash. Running the collector creates an output file (*Sequence.xml*) in *TestResults*, which captures the order of execution of the test before the crash.
 
-```xml
-<DataCollector friendlyName="blame" enabled="True">
-</DataCollector>
+You can run blame in 3 different modes: 
+- Enabling just the sequence file, but not collecting dumps.
+- Enabling crash dump, to create a dump when testhost crashes.
+- Enabling hang dump, to create a dump when test does not finish before given timeout.
```
Comment (Jak-MS) [inline docs/test/configure-unit-tests-by-using-a-dot-runsettings-file.md:220]:
```suggestion
- Enabling just the sequence file, but not collecting dumps
- Enabling crash dump, to create a dump when testhost crashes
- Enabling hang dump, to create a dump when test does not finish before given timeout
```

**Pull request merged**